.github/CODEOWNERS

```diff
@@ -664,6 +664,10 @@
 # TODO(#10538): Revert ownership to @vojtechjelinek after 2020-09-06.
 /package.json @seanlip
 # TODO(#10538): Revert ownership to @vojtechjelinek after 2020-09-06.
+/requirements.txt @seanlip
+# TODO(#10538): Revert ownership to @vojtechjelinek after 2020-09-06.
+/requirements.in @seanlip
+# TODO(#10538): Revert ownership to @vojtechjelinek after 2020-09-06.
```

Comment:
I'm back from my vacation, youu can assign these to me and remove the TODO comments.

Reply:
Done.